geode-core/src/main/java/org/apache/geode/cache/EvictionAttributes.java

```diff
@@ -423,7 +423,12 @@ public static EvictionAttributes createLRUMemoryAttributes(ObjectSizer sizer,
 
   /**
    * The unit of this value is determined by the definition of the {@link EvictionAlgorithm} set by
-   * one of the creation methods e.g. {@link EvictionAttributes#createLRUEntryAttributes()}
+   * one of the creation methods e.g. {@link EvictionAttributes#createLRUEntryAttributes()}.
```

Comment:
I recommend using <br> at the end of lines where we want to see a line feed or use <li><ul> to transform the "If the algorithm" lines into a list with bullets.

Reply:
done